__tests__/endpoints.test.js

```diff
@@ -37,5 +37,36 @@ describe("/api", () => {
         })
      })
 })
+describe("/api/articles/:article_id", () => {
+    test(`Get 200 an an article object containing the following properties: 
+    author, title, article_id, body, topic, created_at, votes, article_img_url`, () => {
+        return request(app).get("/api/articles/2").expect(200)
+        .then((res) => {
+            const  {article } = res.body
```

Comment:
While this test is fine, I think it could be more specific. We know exactly what article 2 looks like and therefore could test the values returned as well as the keys and data types